contrib/graphite/tests/test_blueflood.py

```diff
@@ -0,0 +1,254 @@
+from unittest import TestCase, skip
```

Comment:
Unless the tests already existed as python, I would have suggested using groovy since the team is more familiar with that, but this is fine.